### ISSUE TITLE
[DRG] Lance Charge to Battle Litany feature does not appear to be working

### ISSUE THREAD

**Issue by NikaLeets** [Nik-Potokar/XIVSlothCombo#1580]
At least below lvl 60, the Lance Charge to Battle litany feature does not work at all for me.  Sloth version 3.2.0.2 Adding the actions separately on the hotbars work fine for me.

**Comment by Kagekazu** [comment]
It swaps Lance charge to Battle litany for me.

Can u update to the latest version and see if they still dont go off?

**Comment by NikaLeets** [comment]
Scan forward to 1:00 in the video here.  This was taken on 3.2.0.3 and lance charge was not swapping for me.  I think I how the current settings at the end of the video as well.  I wonder if it's a setting incompatibility issue?

https://drive.google.com/file/d/12N3nf6vG56Wchw-HgIrretOdwv6jA3tU/view?usp=sharing

**Comment by Kagekazu** [comment]
https://github.com/user-attachments/assets/41034828-8f9b-4d21-9a81-c7f133fc5db1

works for me.

**Comment by NikaLeets** [comment]
Ok, I think I have discovered the root cause of this issue.  The program is I ONLY play DRG in PotD, so I haven't done the class quests yet.  When I went and did the lvl 52 class quest to learn battle litany it now started working correctly.  

Note:  Prior to dawntrail, this was not an issue.  Has the method Sloth uses to detect if an ability is learned changed for dawntrail?  Anyway, it's probably something to keep in mind that abilities probably need to be learned outside of deep dungeons now for automaton to work for those abilities inside deep dungeon.

I'll do the rest of the class quests and confirm that this hopefully solves a number of issues.
